### DISCUSSION TITLE
Allow ignoring/excluding a css .class from being transformed by CSS Module

### DISCUSSION THREAD

**Discussion by JuanmaMenendez** [vercel/next.js#19680] [Ideas]
# Feature request
Allow ignoring/excluding a css .class from being transformed by CSS Module

## Is your feature request related to a problem? Please describe.

I need to have normal CSS classes with specifics names (without being processed by CSS Modules).
I cannot have those classes in a global scope (imported in _app.tsx)

## Describe the solution you'd like

CSS modules currently support the use of [:global or :local](https://github.com/css-modules/css-modules#exceptions) for those cases, but when we try to use it on nextjs we get the error:
_"Syntax error: Selector ":global" is not pure (pure selectors must contain at least one local class or id)"_

Nextjs needs to support :global and :local from CSS Modules

## Additional context

Example to reproduce

in myComponent.module.css

```
:global {
.geosuggest__suggests--hidden {
  max-height: 0;
  overflow: hidden;
  border-width: 0;
  background: blue !important;
}

.geosuggest__item--active {
  background: red !important;
  color: #fff;
}
}
```

**Comment by sujin2f** [comment]
It's outdated but FYI, just remove `.module` from your file name. `.module` automatically adds unique identifier into class names.
